client/client_test.go

```diff
@@ -187,3 +188,30 @@ func TestClient(t *testing.T) {
 		})
 	})
 }
+
+func TestGetActionTimeout(t *testing.T) {
+	type testStartActionReq struct {
+		req             *agentpb.StartActionRequest
+		expectedTimeout *duration.Duration
+	}
+
+	testCases := []*testStartActionReq{
+		{
+			req:             &agentpb.StartActionRequest{Timeout: ptypes.DurationProto(0 * time.Second)},
+			expectedTimeout: ptypes.DurationProto(10 * time.Second),
+		},
+		{
+			req:             &agentpb.StartActionRequest{Timeout: nil},
+			expectedTimeout: ptypes.DurationProto(10 * time.Second),
+		},
+		{
+			req:             &agentpb.StartActionRequest{Timeout: ptypes.DurationProto(15 * time.Second)},
+			expectedTimeout: ptypes.DurationProto(15 * time.Second),
+		},
+	}
+
+	for _, tc := range testCases {
+		client := New(nil, nil, nil)
```

Comment:
I think we could use a subtest (`t.Run`) there; otherwise, it is not clear if the first or the second test case fails – expected values are the same